src/getCoinCombination.test.js

```diff
@@ -7,4 +7,59 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('combination of coins for 1 cent', () => {
```

Comment:
I would suggest to use a bit more specific test name:
`should return '1' pennie when '1' is entered`

Reply:
fixed